controller_manager/controller_manager/spawner.py

```diff
@@ -160,6 +160,7 @@ def main(args=None):
     switch_timeout = args.switch_timeout
     strictness = SwitchController.Request.STRICT
     unload_controllers_upon_exit = False
+    node: Node | None = None
```

Comment:
I might need to change this line, since writing union type annotations like this is supported since Python3.10, and RHEL 9 targets Python3.9.

I could either remove the annotation or replace it with `Optional[Node]`. depending on what is preferred.

Reply:
```suggestion
    node = None
```

Why not just initialize it with None?

Reply:
Yeah, type annotations might make it more clear what's happening, so I've gotten into the habit of adding them.